src/live-reload.ts

```diff
@@ -0,0 +1,17 @@
+const RELOAD_ADDRESS = 'ws://localhost:18888'
+
+function setupLiveReloadConnections(address: string): void {
+  let connection = new WebSocket(address)
+  connection.onclose = () => {
+    connection = new WebSocket(address)
+  }
+  connection.onmessage = () => {
+    chrome.runtime.reload()
+  }
+}
+
+chrome.management.getSelf((self) => {
+  if (self.installType === 'development') {
+    setupLiveReloadConnections(RELOAD_ADDRESS)
+  }
```

Comment:
I would also set the `process.env.ENVIRONMENT` by webpack to use that here because extension devs sometimes leave the built version of the extension running and active. 
In case like that, if the webpack websocket server is not running will it throw error?